handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- name: Reloading APT timer
```

Comment:
I would prefer renaming it to what it actually does i.e. `name: reload systemd` so it is clear that this could not only affect the apt timer.

Reply:
Ok